### PR TITLE
Add table 'Recent companies' on dashboard

### PR DESCRIPTION
**Pull request by sviatoslav-krupa**
**CHECK LIST**
- [ ] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs
- [ ] screenshot of rails_best_practices and other tools added
- [x] screenshot of feature is added if task has UI

new_recent_companies
pages

### CODE REVIEW

File: app/views/account/dashboard/_recent_companies.html.slim [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.card[style="height: 395.25px;"]
```
Comment (romaslmd) [inline app/views/account/dashboard/_recent_companies.html.slim]: avoid css in html

File: app/controllers/account/dashboard_controller.rb [hunk 2]
```diff
@@ -1,8 +1,11 @@
 class Account::DashboardController < ApplicationController
+  LIMIT_OF_RECENT_COMPANIES = 5
```
Comment (romaslmd) [inline app/controllers/account/dashboard_controller.rb]: ???

File: app/controllers/account/dashboard_controller.rb [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
+
   before_action :authenticate_user!
 
   def index
     @chart_value = current_companies
     @count_of_pages = CompanyPagesAnalytics.count_pages(current_tenant)
+    @recent_companies = current_companies.order(created_at: :desc).limit(10)
```
Comment (romaslmd) [inline app/controllers/account/dashboard_controller.rb]: prefer https://apidock.com/rails/ActiveRecord/QueryMethods/reorder

File: app/views/account/dashboard/_recent_companies.html.slim [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+          tr
+            th Company name
+            th Domain
+            th Date of creation
+        tbody
+          - recent_companies.each do |company|
+            tr
+              td.text-truncate = link_to company.name, account_company_path(company)
+              td.text-truncate = company.domain
+              td.text-truncate = company.created_at.strftime('%d/%m/%Y  %H:%M')
```
Comment (romaslmd) [inline app/views/account/dashboard/_recent_companies.html.slim]: `I18n.l`

File: app/views/account/dashboard/_recent_searches.html.slim [hunk 5]
```diff
@@ -0,0 +1,95 @@
+.card[style="height: 395.25px;"]
```
Comment (romaslmd) [inline app/views/account/dashboard/_recent_searches.html.slim]: fixed height? move to a css or better remove at all and use some lib for example https://github.com/liabru/jquery-match-height

File: app/controllers/account/dashboard_controller.rb (hunk 2 above)
Comment (sviatoslav-krupa) [reply app/controllers/account/dashboard_controller.rb]: my mistake
something wrong during rebasing

File: app/controllers/account/dashboard_controller.rb [hunk 6]
```diff
@@ -1,8 +1,13 @@
 class Account::DashboardController < ApplicationController
+  LIMIT_TO_SHOW = 5
+
   before_action :authenticate_user!
 
   def index
     @chart_value = current_companies
     @count_of_pages = CompanyPagesAnalytics.count_pages(current_tenant)
+    @recent_companies = current_companies
```
Comment (igorkasyanchuk) [inline app/controllers/account/dashboard_controller.rb]: create scope "recent" and use

File: app/views/account/dashboard/_recent_searches.html.slim [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+        li
+          a[data-action="expand"]
+            i.ft-maximize
+  .card-content
+    .table-responsive
+      table#recent-orders.table.table-hover.mb-0.ps-container.ps-theme-default
+        thead
+          tr
+            th
+              | Serach ID
```
Comment (igorkasyanchuk) [inline app/views/account/dashboard/_recent_searches.html.slim]: why the whole table is hard-coded? can you use real values?

File: app/controllers/account/dashboard_controller.rb [hunk 8]
```diff
@@ -1,8 +1,14 @@
 class Account::DashboardController < ApplicationController
+  LIMIT_TO_SHOW = 5
+
   before_action :authenticate_user!
 
   def index
     @chart_value = current_companies
     @count_of_pages = CompanyPagesAnalytics.count_pages(current_tenant)
+    @recent_companies = current_companies.recent.limit(LIMIT_TO_SHOW)
+    @recent_pages = current_tenant.pages.where(company_id: current_companies.pluck(:id))
```
Comment (igorkasyanchuk) [inline app/controllers/account/dashboard_controller.rb]: bad code. do without pluck. with joins+where

Comment (sviatoslav-krupa) [reply app/controllers/account/dashboard_controller.rb]: Sorry, I forgot that I have has many through association, so I can call pages directly without 'where'

**Pull request merged**